### ISSUE TITLE
Withdraw to external wallet does not close trade

### ISSUE THREAD

**Issue by viperperidot** [bisq-network/bisq#5523]
### Description
After completing a trade and withdrawing funds to an external wallet, the trade does not close and get moved to the history tab. In order to do this you have to click 'keep funds in bisq wallet'. At this point the funds have already been sent to the external wallet so they do not stay in the bisq wallet but this closes the trade and moves it to the history tab. I noticed this bug was introduced a few versions ago.

#### Version
1.6.4

### Steps to reproduce
1. Complete trade
2. Withdraw funds to external wallet
3. Notice that the trade is not closed or moved to the history tab

### Expected behaviour
Trade closes and gets moved to history tab like how it works when you click 'keep funds in bisq wallet'.

### Actual behaviour
Trade stays open

#### Device or machine
Linux - Ubuntu

**Comment by BtcContributor** [comment]
I just tried this scenario on a Windows 10 machine and it seems to work as expected.
So I can't reproduce that but maybe there is something I missed.

**Comment by ghost** [comment]
I also tried withdraw to external wallet; it worked as expected.  (Ubuntu Linux)

**Comment by viperperidot** [comment]
Hmmm, that is very strange because it happens to me every trade...

**Comment by ripcurlx** [comment]
> Hmmm, that is very strange because it happens to me every trade...

could you please look at the logs when you do that. In case there is a silent exception thrown.

**Comment by cd2357** [comment]
@viperperidot Have you tried waiting for a few confirmations? Maybe it needs that before closing trade.

**Comment by bisqubutor** [comment]
> 
> 
> Hmmm, that is very strange because it happens to me every trade...

That's correct. The 'keep funds in bisq wallet' Button does only move trade to trade history but do not touch wallet at all.
The deposit payout already started with pressing "confirm payment received" and this already gave you your funds in bisq wallet.

**Comment by viperperidot** [comment]
Next time I will try to wait until it gets confirmed and see if that works, although I think I remember in previous versions the trade would still close right away when sending to external wallet. I will also check the logs next time too. It may be a little while before I do another trade but I will post an update here if I find more information on this.

**Comment by ripcurlx** [comment]
> Next time I will try to wait until it gets confirmed and see if that works, although I think I remember in previous versions the trade would still close right away when sending to external wallet. I will also check the logs next time too. It may be a little while before I do another trade but I will post an update here if I find more information on this.

I also don't think it should be related to a block confirmation, but let's see the log to be sure.

**Comment by RequestPrivacy** [comment]
I can confirm @viperperidots experience. 

Withdrawn to external wallet some time ago, 4 digit block confirmation but it still remains in the open trade tab. Trying to withdraw "a second time" produces following warning screen:

![Screenshot from 2021-08-12 12-04-24](https://user-images.githubusercontent.com/72392544/129179324-73dd5054-0450-4e49-9eec-2464793e9b7c.png)

Off course fees aren't the problem - the UTXO just isn't anymore in the bisq wallet.

Also on a Linux machine, Bisq v1.7.2

**Comment by kev-vl** [comment]
Can confirm @viperperidot's experience.

**Version**

1.8.0

**Device or machine**

macOS 12 - Intel
